packages/example/package.json

```diff
@@ -17,18 +16,14 @@
     "predeploy": "yarn build",
     "deploy": "gulp deploy"
   },
-  "files": [
-    "cypress",
-    "lib"
-  ],
   "dependencies": {
     "bluebird": "3.5.3",
     "glob": "7.1.3"
   },
   "devDependencies": {
     "chai": "3.5.0",
     "cross-env": "6.0.3",
-    "cypress-example-kitchensink": "https://github.com/cypress-io/cypress-example-kitchensink.git#994a943b76f1186c10ce083f48266cee2f76aeb4",
+    "cypress-example-kitchensink": "https://github.com/cypress-io/cypress-example-kitchensink.git#bd93f68568bc0a412b2c4dd2ddbc3b7cb89e5799",
```

Comment:
crumbs, do i need to release a 12.0.1 for this?

Reply:
I don't think so. its out test dependencies & isn't shipped with Cypress.

Reply:
Why is it a dev dependency?

Reply:
Why is this in devDeps though. 🤔